src/util/dev/builder-cache.ts

```diff
@@ -168,8 +242,8 @@ export async function installBuilders(
         'add',
         '--exact',
         '--no-lockfile',
-        buildUtils,
-        ...packages.filter(p => p !== '@now/static')
+        '--non-interactive',
```

Comment:
Should we add `--non-interactive` to fargate too?

Reply:
Probably